README.md

```diff
@@ -75,12 +80,26 @@ Build and deploy cloudbeat on your local kind cluster:
 just build-deploy-cloudbeat
 ```
 
+Or without certificate
+
+```zsh
+just build-deploy-cloudbeat-nocert
+```
+
 > **Note** By default, cloudbeat binary will be built based on `GOOS` and `GOARCH` environment variables.
 > If you want to build cloudbeat for a different platform you can set them as following:
+>
 > ```zsh
 > # just build-deploy-cloudbeat <Target OS> <Target Arch>
 > just build-deploy-cloudbeat linux amd64
 > ```
+>
+> Or without certificate
+>
+> ```zsh
+> # just build-deploy-cloudbeat-nocert <Target OS> <Target Arch>
+> just build-deploy-cloudbeat-nocert linux amd64
+> ```
```

Comment:
can you remove the part about `<Target OS>` and `GOOS` from the docs? it's no longer relevant